rust/lanzatool/src/cli.rs

```diff
@@ -27,10 +27,14 @@ struct InstallCommand {
     #[arg(long)]
     private_key: PathBuf,
 
+    /// Configuration limit
+    #[arg(long, default_value_t = 1)]
+    configuration_limit: u64,
```

Comment:
You may want to keep this as `usize` to avoid the awkward conversion between `u64` and `usize`.

Reply:
I guess my intuition was that I'd rather have an explicit type than usize. Thinking about it more, I don't really see a good reason for why u64 would be better.